instagram-app/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import './CommentSection.css'
+const CommentSection = ({comments})=>{
+    return (
```

Comment:
Don't forget to use prop-types when you are receiving props in a component.
Be careful you are missing them in most components.